src/oph/ehoks/hoks/vipunen_schema.clj

```diff
@@ -144,7 +144,8 @@
   "Varmistaa, että keskeytymisajanjaksot eivät mene päällekkäin."
   [jaksot]
   (or (<= (count jaksot) 1)
-      (reduce #(if (and (:loppu %1) (.isBefore (:loppu %1) (:alku %2)))
+      (reduce #(if (and (:loppu %1) (.isBefore ^LocalDate (:loppu %1)
```

Comment:
Erillisestä vipunen-schemasta pitäis myös päästä eroon.  Tai siis siitä, että se on tehty kopioimalla koodit.  Mut ei tän PR:n asia.